### PR TITLE
Add support for Choco60

### PR DESCRIPTION
**Pull request by takai**
## Description

Add support for Choco60

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/choco60/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#include QMK_KEYBOARD_H
+
+#define _BASE 0
+#define _FN   1
+
+#define KC_FN MO(_FN)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_BASE] = LAYOUT(
+    KC_ESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINUS, KC_EQUAL, KC_BSLASH, KC_GRAVE, \
```
Comment (fauxpark) [inline keyboards/choco60/keymaps/default/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

File: keyboards/choco60/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# choco60
+
+![choco60](https://keys.recompile.net/images/choco60-main@600w.jpg)
+
+A 60% split keyboard for programmers.
+
+Keyboard Maintainer: [Naoto Takai](https://github.com/takai)
```
Comment (fauxpark) [inline keyboards/choco60/readme.md]:
```suggestion
Keyboard Maintainer: [Naoto Takai](https://github.com/takai)  
```

The double spaces are needed here for GitHub to render this correctly.

File: keyboards/choco60/readme.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Hardware Supported: The Choco60 PCBs, Pro Micro supported
```
Comment (fauxpark) [inline keyboards/choco60/readme.md]:
```suggestion
Hardware Supported: The Choco60 PCBs, Pro Micro supported  
```

**Comment by takai** [comment]
@fauxpark Thank you for the review!

File: keyboards/choco60/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#define _BASE 0
```
Comment (noroadsleft) [inline keyboards/choco60/keymaps/default/keymap.c]: As I said on the Cocoa40 pull request, you could use an enum here in place of Lines 19-20:

```c
enum layer_names {
    _BASE,
    _FN,
};
```

I think it's less important to do when there are only two layers; I usually only insist on it when there are four layers or more.

File: keyboards/choco60/rules.mk [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration(+1000)
```
Comment (drashna) [inline keyboards/choco60/rules.mk]: Does this keyboard have a physical reset button?

```suggestion
BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration
```

Comment (takai) [reply keyboards/choco60/rules.mk]: Thank you for the advice. Choco60 have a physical reset button.

Comment (drashna) [reply keyboards/choco60/rules.mk]: Thanks for confirming! 

Not every board does, and sometimes it's not easy to reach.  This makes things easier, in that case.

**Pull request merged**